lease/lessor.go

```diff
@@ -219,15 +224,27 @@ func (le *lessor) Revoke(id LeaseID) error {
 	le.mu.Unlock()
 
 	if le.rd != nil {
```

Comment:
`if le.rd == nil { return nil }` to pull back the indent for the interesting path?